### ISSUE TITLE
get started tutorial

### ISSUE THREAD

**Issue by bobzhang** [hackwaly/vscode-ocaml#61]
this looks quite promising, is there a get started configuratoin?

**Comment by hackwaly** [comment]
No, but there's my starter project https://github.com/hackwaly/stone/commit/c3f8ed6dcc758c36422e30f341ffd12bedd357c8 you may have interest in.
